### PR TITLE
🚀 RELEASE: v2.0.0

### PR DESCRIPTION
**Pull request by chrisjsewell**
@sphuber, once you do the final `Node.objects` -> `Node.collection` change, I think it makes sense to quickly push out a release today?

It will just make it quicker when it comes to the final v2.0 release

### CODE REVIEW

**Comment by chrisjsewell** [comment]
See: https://aiida--5488.org.readthedocs.build/projects/aiida-core/en/5488/reference/_changelog.html

**Comment by sphuber** [comment]
There are a few more PRs that probably should go in. They shouldn't take long to review and they are important for v2.0

#5487 
#5479 
#5476 
#5468 
#5461 
#5460 
#5457

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,142 @@
 # Changelog
 
+## v2.0.0b2 - 2022-04-12
```
Comment (sphuber) [inline CHANGELOG.md:3]:
```suggestion
## v2.0.0 - 2022-04-27
```

**Review by sphuber — COMMENTED** [review]
Not sure if you were already working on it, but we should also probably still add the PR #5457 #5468 #5478

**Review by sphuber — APPROVED** [review]

**Pull request merged**